.github/workflows/cli.yml

```
@@ -42,52 +42,125 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test:
+  e2e-ubuntu:
```

```suggestion
  jest-ubuntu:
```

Maybe this makes more sense, as playwright is also running E2E tests.